EstateAdministrationUI.Testing/TestData.cs

```diff
@@ -0,0 +1,303 @@
+﻿namespace EstateAdministrationUI.Testing
+{
+    using System;
+    using System.Collections.Generic;
+    using Areas.Estate.Models;
+    using BusinessLogic.Models;
+    using EstateManagement.DataTransferObjects.Responses;
+
+    public class TestData
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5756318)